README.md

```diff
@@ -1,2 +1,73 @@
 # jboss-eap-8-openshift-image
  OpenShift container images for Red Hat JBoss Enterprise Application Platform 8
+
+This project defines Images allowing you to build and deploy EAP 8 applications on the cloud.
+
+EAP 8 S2I (Source-To-Image) builder images:
+
+* `jboss-eap-8-tech-preview/eap8-openjdk11-builder-openshift-rhel8`
+
+* `jboss-eap-8-tech-preview/eap8-openjdk17-builder-openshift-rhel8`
+
+EAP 8 runtime images:
+
+* `jboss-eap-8-tech-preview/eap8-openjdk11-runtime-openshift-rhel8`
+
+* `jboss-eap-8-tech-preview/eap8-openjdk17-runtime-openshift-rhel8`
+
+
+# Relationship with EAP 7.x images
+
+As opposed to the EAP 7.x S2I (Source-To-Image) builder image that is containing an EAP server, the new builder image 
+is a generic builder allowing to build image for any EAP 8 releases.
+
+# EAP 8 - S2I builder image
+
+The S2I builder image contains all you need to execute a Maven build of your project during an S2I build phase on OpenShift.
+
+## S2I build workflow
+
+The builder image expects that, during the Maven build, an EAP 8 server containing the deployment is being provisioned (by default in `<application project>/target/server` directory). This provisioned server 
+is installed by the image in `/opt/server`. Making the generated application image runnable.
+
+In order to provision a server during the build phase you must integrate (generally in a profile named `openshift` profile) an execution of the  [EAP Maven plugin](https://github.com/jbossas/eap-maven-plugin/).
+
+## Using the S2I builder image
+
+The more efficient way to use the EAP 8 S2I builder image to construct an EAP application image is by using [EAP Helm charts](https://github.com/jbossas/eap-charts).
+EAP Helm Charts  are automating the build (on OpenShift) and deployment of your application by the mean of a simple yaml file.
+
+## Galleon feature-pack for EAP 8 on the Cloud
+
+The [EAP cloud feature-pack](https://github.com/jbossas/eap-cloud-galleon-pack) contains all the cloud specifics that were contained in the EAP 7.x image.
+This feature-pack has to be provisioned along with the EAP 8 `org.jboss.eap:wildfly-ee-galleon-pack` feature-pack. 
+
+For more information on the EAP cloud feature-pack features, check [this documentation](https://github.com/jbossas/eap-cloud-galleon-pack/blob/main/README.md).
+
+## Backward compatible S2I workflow
+
+In case you want to keep your existing project that used to work with the legacy EAP 7.x S2i builder image, you can use a set of environment variables 
+to initiate a server provisioning prior to execute the Maven build of your application:
+
+* `GALLEON_PROVISION_FEATURE_PACKS`: Comma separated lists of Galleon feature-packs, for example: 
+`GALLEON_PROVISION_FEATURE_PACKS=org.jboss.eap:wildfly-ee-galleon-pack,org.jboss.eap.cloud:eap-cloud-galleon-pack:8.0.0.Final-redhat-00001` 
```

Comment:
do you need to specify the `version` of `org.jboss.eap:wildfly-ee-galleon-pack`?